docs/spark/tutorials/databricks-deployment.md

```diff
@@ -143,6 +143,25 @@ Next, you publish the *mySparkApp* created in the [.NET for Apache Spark - Get S
    zip -r publish.zip .
    ```
 
+3. If you have any user-defined functions in your app, the app assemblies, such as DLLs that contain user-defined functions along with their dependencies, need to be placed in the working directory of each *Microsoft.Spark.Worker*.
+
+    Upload your application assemblies to your Databricks cluster:
+
+    ```console
+    cd <path-to-your-app-publish-directory>
+    databricks fs cp <assembly>.dll dbfs:/apps/dependencies
```

Comment:
Is the `apps/dependencies` directory a default in Databricks or is it something that users have to create? If the latter, it'd be good to mention that.